### PR TITLE
graphql, internal: fix typos in comments

### PR DESCRIPTION
**Pull request by DavidRomanovizc**
#### What was wrong?
There are typos in the code comments and in other files

#### How was it fixed?
Fixed them

#### Cute Animal Picture
image

### CODE REVIEW

**Comment by karalabe** [comment]
Quote a number of these were blind replaces without realising the originals were correct.

File: les/txrelay.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ func (ltrx *lesTxRelay) registerPeer(p *serverPeer) {
 	ltrx.lock.Lock()
 	defer ltrx.lock.Unlock()
 
-	// Short circuit if the peer is announce only.
+	// Short circuit if the peer is announced only.
```
Comment (holiman) [inline les/txrelay.go:59]: revert

File: les/client.go [hunk 2]
```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package les implements the Light Ethereum Subprotocol.
+// Package less implements the Light Ethereum Subprotocol.
```
Comment (holiman) [inline les/client.go:17]: revert

File: internal/shutdowncheck/shutdown_tracker.go [hunk 3]
```diff
@@ -34,7 +34,7 @@ type ShutdownTracker struct {
 }
 
 // NewShutdownTracker creates a new ShutdownTracker instance and has
-// no other side-effect.
+// no other side effect.
```
Comment (holiman) [inline internal/shutdowncheck/shutdown_tracker.go:37]: revert

File: internal/jsre/pretty.go [hunk 4]
```diff
@@ -208,7 +208,7 @@ func (ctx ppctx) fields(obj *goja.Object) []string {
 		key := SafeGet(obj, k)
 		if key == nil {
 			// The value corresponding to that key could not be found
-			// (typically because it is backed by an RPC call that is
+			// typically because it is backed by an RPC call that is
```
Comment (holiman) [inline internal/jsre/pretty.go:211]: revert

File: internal/jsre/jsre.go [hunk 5]
```diff
@@ -220,7 +220,7 @@ loop:
 	}
 }
 
-// Do executes the given function on the JS event loop.
+// Do execute the given function on the JS event loop.
```
Comment (holiman) [inline internal/jsre/jsre.go:223]: revert

File: internal/debug/trace.go [hunk 6]
```diff
@@ -48,7 +48,7 @@ func (h *HandlerT) StartGoTrace(file string) error {
 	return nil
 }
 
-// StopTrace stops an ongoing trace.
+// StopGoTrace StopTrace stops an ongoing trace.
```
Comment (holiman) [inline internal/debug/trace.go:51]:
```suggestion
// StopGoTrace stops an ongoing trace.
```

File: internal/build/gotool.go [hunk 7]
```diff
@@ -74,7 +74,7 @@ func (g *GoToolchain) Install(gobin string, args ...string) *exec.Cmd {
 	// Ensure GOPATH is set because go install seems to absolutely require it. This uses
 	// 'go env' because it resolves the default value when GOPATH is not set in the
 	// environment. Ignore errors running go env and leave any complaining about GOPATH to
-	// the install command.
+	// the installation command.
```
Comment (holiman) [inline internal/build/gotool.go:77]: revert

**Review by holiman — COMMENTED** [review]
Please revert the marked changes

**Comment by deep709** [comment]
Always best innovative ideas .

File: internal/web3ext/web3ext.go [hunk 8]
```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-// package web3ext contains geth specific web3.js extensions.
+// Package web3ext package web3ext contains geth specific web3.js extensions.
```
Comment (holiman) [inline internal/web3ext/web3ext.go:17]:
```suggestion
// Package web3ext contains geth specific web3.js extensions.
```

**Review by holiman — APPROVED** [review]
LGTM, thanks

**Pull request merged**